### DISCUSSION TITLE
Keep dap REPL output when debugging session fails to launch

### DISCUSSION THREAD

**Discussion by coinator** [mfussenegger/nvim-dap#259] [Q&A]
If the debugging session will fail to launch due to misconfigurations from my behalf (e.g. launching a debugpy session from a parent directory of the script). The dap REPL will display an error message and then the whole layout will close almost instantly.

Is there any way to make the layout persist or alternatively store the REPL output?

**Comment by mfussenegger** [comment]
Are you using nvim-dap-ui?

nvim-dap by itself doesn't change the layout if the session fails to launch and it keeps the REPL output until you start a new session.

**Reply by coinator** [reply]
Oh yes. It's `nvim-dapui` causing me the confusion. It is also the one clearing the REPL output.

**Reply by mfussenegger** [reply]
Could you bring this up in https://github.com/rcarriga/nvim-dap-ui then?
